src/registry_schemas/schemas/party.json

```diff
@@ -12,7 +12,8 @@
                     "title": "The type of party",
                     "enum": [
                         "organization",
-                        "person"
+                        "person",
+                        "businessOrCorporation"
```

Comment:
organization covers this, or at least that is the intent.
people or org... or you could call is a business if you wanted to change it everywhere.

Reply:
removed businessOrCorporation